Lesson5/Program.cs

```diff
@@ -0,0 +1,14 @@
+﻿using ManagerScannerLibrary;
+using ScannerLibrary;
+
+var scanner = new ScannerService();
+
+var manager = new ManagerScannerService(scanner)
+{
+    SourceFileName = "Text.txt",
+    TargetFileName = "Result.txt"
+};
+manager.SetupScan(new TxtFormat());
+await manager.RunScanner().ConfigureAwait(false);
```

Comment:
```suggestion
await manager.RunScanner().ConfigureAwait(true);
```
Если рассматривать консольное приложение как интерфейс пользователя, то в данном месте надо сохранять поток UI (хоть это в консоли и не имеет смысла).